configure

```diff
@@ -0,0 +1,999 @@
+#!/bin/sh
+
+#  !---------------------------------------------------------------------!
+#  ! Written by Madu Manathunga on 07/09/2020                            !
+#  !                                                                     !
+#  ! Copyright (C) 2020-2021 Merz lab                                    !
+#  ! Copyright (C) 2020-2021 Götz lab                                    !
+#  !                                                                     !
+#  ! This Source Code Form is subject to the terms of the Mozilla Public !
+#  ! License, v. 2.0. If a copy of the MPL was not distributed with this !
+#  ! file, You can obtain one at http://mozilla.org/MPL/2.0/.            !
+#  !_____________________________________________________________________!
+#
+#  !---------------------------------------------------------------------!
+#  ! This source code sets up the environment for QUICK compilation and  !
+#  ! writes input file for make.                                         !
+#  !---------------------------------------------------------------------!
+
+#  !---------------------------------------------------------------------!
+#  ! Set version                                                         !
+#  !---------------------------------------------------------------------!
+QUICK_VERSION='QUICK-20.06'
+QUICK_HOME=`pwd`
+cdate=`date +'%m/%d/%Y'`
+ctime=`date +'%r'`
+
+echo ""
+echo -e "\033[92mConfiguring $QUICK_VERSION installation started on $cdate at $ctime..\033[0m" 
```

Comment:
When using `/bin/sh` on a MacOS (I have Mojave), the `echo` command literally prints `-e`:
```
andi@blitz:~/projects/quick/quick>./configure
-e Configuring QUICK-20.06 installation started on 07/29/2020 at 01:11:36 AM..
-e                                                                                                                                        
  Use this script as ./configure [flags] compiler                           
```
Also, can we keep it default color (i.e. black for me)? The neon green looks quite harsh on my eyes, in particular on white background. Darker green (color 32) might work. But then again, it will depend on the color people have in their terminal. I'd vouch for not using colot.

Reply:
Sure. I can remove all the colors.